system/cache/README.md

```diff
@@ -0,0 +1,63 @@
+# Cache
+It is a caching library for the wrappers. Wrappers can store and retrieve data on cache using this library.
+
+This library has been implemented in easily extensible way. It provides a common caching interface that can be shared across various caching implementations. We have already implemented the in-memory cache implementation and will be adding more such implementations. 
```

Comment:
I think here would be cool if we name some of these implementations we are planning to do, my first thoughts can be using local-storage or ceramic I guess?